docs/getting-started/get-started-nuxt-3.md

```diff
@@ -74,94 +68,218 @@ Kendo UI for Vue includes [four artfully designed themes](slug:themesandstyles)
     ```sh
       npm install --save @progress/kendo-theme-default
     ```
-    <!---
-    ```sh
-    yarn add --save @progress/kendo-theme-default
-    ```
-    --->
 
 1. In the `nuxt.config.ts` file, import the CSS files provided by the installed theme package:  
 
-    ```js
-      import '@progress/kendo-theme-default/dist/all.css';
+    ```ts
+      export default defineNuxtConfig({
+        css: [
+          '@progress/kendo-theme-default/dist/all.css',
+        ],
+      })
     ```
 
+>To add any custom styles to your app, insert a `<styles>` tag in the `src/App.vue` file and place your styles there.
 
 ## Add a Vue Data Grid Component
 
-
 1. Now that you've installed all required packages, you are ready to add the Kendo UI for Vue Data Grid to the application.
 
-```sh
-npx nuxi add page KendoGrid
-```
-
-1. In the `pages/KendoGrid.vue` file, add a `<script>` block and import the Grid and its data:
+    ```sh
+    npx nuxi add page KendoGrid
+    ```
 
+1. Add a `<script>` block to the `pages/KendoGrid.vue` file, import the Grid and its data. In addition, the `process` function from the [Data Query](https://www.telerik.com/kendo-vue-ui/components/dataquery/) package will allow you to apply data operations like sorting, paging, and filtering.
 
-```js
-import { productsData } from '../appdata/products';
-import { process, DataResult, State, CompositeFilterDescriptor, SortDescriptor } from '@progress/kendo-data-query';
-import { Grid as grid } from '@progress/kendo-vue-grid';
-```
+    ```js
+    <script>
+        import { productsData } from '../appdata/products';
+        import { process, type DataResult, type State, type SortDescriptor } from '@progress/kendo-data-query';
+        import { Grid as grid } from '@progress/kendo-vue-grid';
+    </script>
+    ```
 
 1. Add a `<template>` block with a simple heading and create a Data Grid. Bind it to the `products` data:
```

Comment:
In my version of the app, the template block exists and has some content. Should we add the new content in the existing block?

```
<template>
  <div>
    Page: KendoGrid
  </div>
</template>
```